drupal_ti/before/before_script.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Add an optional statement to see that this is running in Travis CI.
+echo "running drupal_ti/before/before_script.sh"
+
+set -e $DRUPAL_TI_DEBUG
+
+# Ensure the right Drupal version is installed.
+# The first time this is run, it will install Drupal.
+# Note: This function is re-entrant.
+drupal_ti_ensure_drupal
+
+# Change to the Drupal directory
+cd "$DRUPAL_TI_DRUPAL_DIR"
+
+composer require "mtdowling/cron-expression:1.2.0"
```

Comment:
FYI: For whatever reason, Drupal TI didn't expose the composer dependencies to PHP Unit inside travis, so this library couldn't be found. So I had to force the issue here. It's not exactly ideal, but only affects Travis, so meh...